distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,855 @@
+|"admin"
+
+|===
+
+.Example: Notification Message
+[source,json,linenums]
+----
+"data": {
+	"application": "Downloads",
+	"title": "Product retrieval successful",
+	"message": "The requested product was retrieved successfully
+		and is available for download.",
+	"id": "27ec3222af1144ff827a351b1962a236",
+	"timestamp": "1403734355420",
+	"user": "admin"
+}
+----
+
+====== Notification Operations Channel
+
+A notification operation is performed by publishing a list of commands to the CometD endpoint at `/notification/action`
+
+.Operation Format
+[cols="1m,2,1,1,2,2" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+|Comments
+
+|action
+|Type of action to request
+|String
+|Any
+|"remove" (Currently only used action)
+|If a client publishes with the `remove` action, it removes the notification with the given id from the persistence store.
+
+|id
+|ID of the notification to which the action relates
+|String
+|Any
+|"27ec3222af1144ff827a351b1962a236"
+|This is the id of the notification
+
+|===
+
+.Example: Notification Operation Request
+[source,json,linenums]
+----
+"data": [ {
+	"action": "remove",
+ 	"id": "27ec3222af1144ff827a351b1962a236"
+} ]
+----
+
+====== Persistence of Notifications
+
+Notifications are persisted between sessions, however due to the nature of CometD communications, they will not be visible at first connection/subscription to ``/${ddf-branding-lowercase}/notifications/**``.
+
+In order to retrieve notifications that were persisted or may have occurred since the previous session a client simply must publish an empty json message, `{}` to `/${ddf-branding-lowercase}/notifications`.
+This will return all existing notifications to the user.
+
+====== Activity Events Channel
+
+To receive all activity updates, follow the instructions at <<asynchronous_search_subscribing, Subscribing to Notifications>> and subscribe to `/ddf/activities/\*\*`
+
+Activity update messages follow a specific format when they are published to the activities channel.
+These messages contain a data map that encapsulates the activity information.
+
+.CometD Activity Format
+[cols="1m,3,1,1,1" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+
+|category
+|Category of the activity
+|String
+|Any String
+|"Product Retrieval"
+
+|id
+|ID that uniquely identifies the activity that sent out the update. Not required
+to be unique per update.
+|String
+|Any String
+|"b72ccdf6-8ca7-4f53-a0f6-b0ad264393b0"
+
+|message
+|User-readable message that explains the current activity status
+|String
+|Any String
+|"Resource retrieval downloading."
+
+|operations
+|Map of operations that can be performed on this activity
+|JSON Map
+|A map of keys with populated values (that evaluate to 'true' rather than 'null' 'undefined' or 'false') These operations and their values can be used by clients to communicate back to the server by sending a message back on the same channel.
```

Comment:
The _Possible Values_  and _Example Values_ are unreadable in the PDF version. Should we consider moving away from the table format for this type of information? Or put that information in a separate section and refer to it from here?

Also, the _(that evaluate to 'true' rather than 'null' 'undefined' or 'false')_ isn't clear to me. @Lambeaux, any way we can improve that?